custom_components/hacs/update.py

```diff
@@ -148,9 +149,15 @@ async def async_release_notes(self) -> str | None:
                 self.repository.data.last_version = next(iter(self.repository.data.published_tags))
 
         release_notes = ""
-        if len(self.repository.releases.objects) > 0:
-            release = self.repository.releases.objects[0]
-            release_notes += release.body
+        if any([r for r in self.repository.releases.objects if r.tag_name == self.installed_version]):
+            new_releases = dropwhile(lambda r: r.tag_name != self.installed_version, reversed(self.repository.releases.objects))
```

Comment:
```suggestion
            new_releases = dropwhile(lambda r: r.tag_name != self.installed_version, self.repository.releases.objects)
```
It should not be reversed to match the format of add-ons (and the general usage of a potential CHANGELOG.md file)
<https://changelog.md/links>

Reply:
actually we need to keep the reverse here (but we'll re-reverse right after to keep the order similar to changelog order)

Reply:
You only need to do that because of the `dropwhile` usage, if you drop that, and instead `break` when you hit the current version that is not needed.

Reply:
order is now matching changelog files